openapi-spec/swagger.yaml

```diff
@@ -1480,11 +1490,33 @@ paths:
               schema:
                 type: array
                 items:
-                  $ref: '#/components/schemas/FileShareSnapshotSpec'
+                  $ref: '#/components/schemas/FileShareSnapshotRespSpec'
+              examples:
+                response:
+                  value:
+                  - id: 405ded84-b6ff-4ac6-8a3c-cb0e82dcf372
+                    createdAt: 2019-12-04T13:23:13
+                    updatedAt: 2019-12-04T13:31:03
+                    tenantId: e93b4c0934da416eb9c8d120c5d04d96
+                    userId:
+                    fileshareId: eedfbfcf-de50-4476-a2b6-058e6bfa3558
+                    name: snaptest
+                    description: Testing file share snapshot
+                    shareSize: 1
+                    snapshotSize: 1
+                    status: available
+                    poolId: 52599613-6f64-5aae-9819-7d2a2c790629
+                    protocols: [nfs]
```

Comment:
is protocols returned by api?

Reply:
Yes

Reply:
this is snapshot not fileshare

Reply:
yes done changes, poolId is also not required i removed that as well